test/test_linalg_skeletonization.py

```diff
@@ -341,9 +343,20 @@ def run_skeletonize_by_proxy(actx, case, resolution,
     return err_f, (places, mat)
 
 
-@pytest.mark.parametrize("case", SKELETONIZE_TEST_CASES)
+@pytest.mark.parametrize("case", [
+    # NOTE: skip 2d tests, since they're better checked for convergence in
+    # `test_skeletonize_by_proxy_convergence`
+    # SKELETONIZE_TEST_CASES[0], SKELETONIZE_TEST_CASES[1],
+    SKELETONIZE_TEST_CASES[2],
+    ])
 def test_skeletonize_by_proxy(actx_factory, case, visualize=False):
-    """Test single-level level skeletonization accuracy."""
+    r"""Test single-level level skeletonization accuracy. Checks that the error
+    satisfies :math:`e < c \epsilon_{id}` for a fixed ID tolerance and an
+    empirically determined (not too huge) :math:`c`.
+
+    NOTE: this test is mostly
```

Comment:
This looks

Reply:
Fixed in https://github.com/inducer/pytential/compare/440161e74f559e51120245ce59f8c1a6c4f15d77..0617bdafd0a7a37c2e162b752371ab908e456ad5!